test/constants_test.go

```diff
@@ -15,7 +15,7 @@ const (
 	tipSetGeneratorSeed = 0x264803e715714f95
 
 	latencyAsync         = 100 * time.Millisecond
-	maxRounds            = 10
+	maxRounds            = 20
```

Comment:
Some multi-instance tests now require >15 rounds to complete in their worst case. The slow ones are with fewer participants, where it's more likely to get repeated rounds of no strong quorum observed in PREPARE, and so many commits for bottom.